src/calculateRentalCost.js

```diff
@@ -26,9 +26,19 @@
  *
  * @return {number}
  */
-
 function calculateRentalCost(days) {
-  // write code here
-}
+  const rental = 40;
+  const discountSmall = 20;
+  const discountBig = 50;
+  let price = days * rental;
+
+  if (days >= 3 && days < 7) {
+    price = (days * rental) - discountSmall;
+  } else if (days >= 7) {
+    price = (days * rental) - discountBig;
```

Comment:
You already have a variable for `days * rental`
```suggestion
    return price - discountSmall;
  } else if (days >= 7) {
    return price - discountBig;
```